rule-template/cuevox.yaml

```diff
@@ -190,17 +191,17 @@ actions:
 
                   var executeFunction = function(parameter) {
                       if (!parameter || typeof(parameter) != "object") {
-                          console.debug("Trying to send a command, but no proper object parameter found")
+                          logger.info("Trying to send a command, but no proper object parameter found")
                           return;
                       }
 
                       if (!parameter.items) {
-                          console.debug("Trying to send a command, but no items parameter found")
+                          logger.info("Trying to send a command, but no items parameter found")
```

Comment:
In some statements, you changed not only the "console" to "logger", but also changed the logging level. By accident or on purpose?

Reply:
I was thinking ahead to much trying to reason about what would be relevant to none developers, reverted for now its probably better done in a separate merge.